scripts/src/build.sh

```diff
@@ -36,7 +36,7 @@ gear_test_build() {
 }
 
 node_build() {
-  cargo build -p gear-node "$@"
+  cargo build -p gear-node $@
```

Comment:
we should always use quotes, passing such args in shell

Reply:
Surprisingly, it didn't work for me with the quotes so I removed them. Not so well-versed with shell scripting, it's possible that the reason it didn't work for me doesn't have to do with quotes. Will check again